content/accommodations.md

```diff
@@ -0,0 +1,37 @@
+---
+title: "Accommodations"
+date: 2022-10-15T18:30:00+02:00
+weight: 10
+draft: false
+menu: main
+type: page
+---
+There are many hotels in Delft. Here is a list with possible hotels that aren’t really
+expensive and are easy to travel from to the campus.
+* The student hotel Delft
+  * Prize: 60-95 euros per night per person
```

Comment:
```suggestion
  * Cost: 60-95 euros per night per person
```
But if you prefer "price" over "cost", please write it with a "c" :stuck_out_tongue: